operators/olm-restricted-networks.adoc

```diff
@@ -40,5 +40,13 @@ include::modules/olm-building-operator-catalog-image.adoc[leveloffset=+1]
 * xref:../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[Creating a mirror registry for installation in a restricted network]
 
 include::modules/olm-restricted-networks-configuring-operatorhub.adoc[leveloffset=+1]
+.Additional resources
+
+* xref:../operators/operator_sdk/osdk-generating-csvs.adoc#olm-enabling-operator-for-multi-arch_osdk-generating-csvs[Multi-arch images support on {product-title}]
```

Comment:
I'd use the full module title, "Enabling your Operator for multiple architectures and operating systems"

But it seems like the audiences are different for this assembly and the linked assembly, so I'm not convinced that the link makes sense as-is. I might break the reference part of the linked module into a separate module and include it somewhere in this assembly.